guides/handGrenades.md

```diff
@@ -0,0 +1,107 @@
+
+> [Frag|Flash|Smoke] out. [Direction]
+
+Examples:
+
+> Frag Out! North!
+> Flash Out! South-East!  
+> Smoke Out! West! 
+
+The call out should not slow down your ability to respond to threats. At a minimum you must always call out "[Grenade Type] Out!"
+
+If the employment of the grenade fails, the thrower should immediately alert the other operators in his vicinity.
+
+Example:
+
+> Bad 'nade!
+> Bad frag! 
+> Bad flash! Get down! 
+
+### Risks of Use
+
+When firing throwing fragmentation grenades, you need to know the impact radius and fragmentation 
+radius to maximize injury on the enemy and reduce friendly injuries.
+
+The approximate lethality of an M67 fragmentation grenade is:
+
+- 5m - Deadly
+- 15m - Casualty radius
+- 230m - Fragmentation radius (shrapnel can disperse up to this far.)
+- 35m - Range an average soldier can throw
+
+Regardless of range there is always a chance of being hit by a piece of shrapnel. 
+Due to this risk, proper employment should always consist of throwing the grenade and immediately seeking cover.
+
+#### Vehicles
+
+It is possible for fragmentation grenades to disable vehicles, however they would need to be right on top of the grenade. 
+It is best practice to exhaust all other means of disabling a vehicle before attempting to throw a frag.
+
+#### Buildings 
+
+Fragmentation and flashbang grenades are most effective when used in enclosed spaces due to the overpressure of the enclosed area 
+along with the shrapnel caused by the fragmentation. A flashbang or fragmentation grenade should be chosen accordingly based on whether
+on-combatants are inside of the room/building. If they are, a flashbang is preferred.
+
+### Grenade Types
+
+- Fragmentation (M67) 
+    - Also known as 'frags'. 
+    - Fuze time of 4 - 5 seconds.
+
+- Flashbang (M84)
+    - Also known as a 'flash'. 
+    - Fuze time of 1 - 2 seconds.
+
+- Smoke (M18)
+    - Comes in a variety of colors. White is typically used as a screen, while colored smokes are used to mark positions.
+    - Fuze time of 1 - 2 seconds.
+    - Lasts 60 seconds
+
+- Chemlights
+    - Deploys immediately 
+    - Used to mark cleared rooms or cleared areas at nighttime. 
+
+### ACE3 Grenades 
+
+The ACE3 grenade system endeavors to make grenades easier to use in Arma3. 
+Two additional keybinds are added in two different modes to assist with grenade usage, as well as a HUD element.
+
+#### ACE3 Grenades Keybinds
+
+- Shift + G
+    - Pulls up the ACE grenade context HUD. You will see the current grenade type, as well as a small keymap along with multiple dots in an arc fashion.
+    - The dots represent the path of the grenade.
+    - If a dot is green it means the grenade is expected to his the ground at the green dot.
+    - If a dot is yellow it means the dot will encounter be going through a closed space (like a window or doorway) and may hit a ledge and change direction.
```

Comment:
```suggestion
    - If a dot is yellow it means the dot will encounter be going through a closed space and may hit a ledge and change direction (like a window or doorway).
```
This is not technically what yellow means, but is a good enough example

Reply:
What does it actually mean? I can change the wording.

Reply:
It basically just means ACE can't determine the outcome, but what you have is fine